conceptual/Npgsql/connection-string-parameters.md

```diff
@@ -57,7 +57,7 @@ Parameter                  | Description
 -------------------------- | ------------------------------------------------------------ | -------
 Max Auto Prepare           | The maximum number SQL statements that can be automatically prepared at any given point. Beyond this number the least-recently-used statement will be recycled. Zero disables automatic preparation. | 0
 Auto Prepare Min Usages    | The minimum number of usages an SQL statement is used before it's automatically prepared. | 5
-Use Perf Counters          | Makes Npgsql write performance information about connection use to Windows Performance Counters. [Read the docs](performance.md#performance-counters) for more info.
+Use Perf Counters          | Makes Npgsql write performance information about connection use to Windows Performance Counters. [Read the docs](performance.md#performance-counters) for more info. | false
```

Comment:
This has been removed for Npgsql 5.0. It has always been .NET Framework only (and Windows-only), and AFAIK very few people have actually used it. I'd be fine with just omitting this from the connection string docs, or we should clearly specify that it has been removed.

Reply:
Note related issue: #48

Reply:
I would add a note as some other entries also contain _"Since 3.1 only"_.  By the way what does the word _"only"_ mean in this context. Do these parameters only exist in v.3.1 or also in later versions like 5.0?

Reply:
I would think that "since" makes it clear that the entries exist also after 3.1, but if you feel that's confusing we can remove the "only". No problem with adding "Removed in 5.0" for the perf counters.